appengine/standard/i18n/i18n_utils.py

```diff
@@ -49,15 +54,15 @@ def _get_plural_forms(js_translations):
         for l in js_translations._catalog[''].split('\n'):
             if l.startswith('Plural-Forms:'):
                 plural = l.split(':', 1)[1].strip()
-                print "plural is %s" % plural
+                print("plural is %s" % plural)
```

Comment:
Prefer string formatter and single quotes:

```
print(`plural is {}`.format(plural))
```

Reply:
[__python/black__](https://github.com/python/black) (think gofmt for Python) is reformatting code to prefer " over ' but I will make the requested change.

Reply:
We'll consider this alternative style; I don't personally have a strong opinion either way. Thank you for the pointers, also thanks a ton for the fixes!

Reply:
Regarding python/black and quotes, according to [the style guide](https://google.github.io/styleguide/pyguide.html) it looks like some teams are using [yapf](https://github.com/google/yapf/).